recipe_sha: 37aaacbfdd150cb6025c53ae4d3b5e23bf3cc74906af91747e649cd79eda8695  # The commit at which this recipe failed to build.
skiplist: true # Set to true to skiplist this recipe so that it will be ignored as long as its latest commit is the one given above.
log: |2-
   - bioconductor-biocgenerics >=0.48.0,<0.49.0
   - r-stringr
   - bioconductor-biostrings >=2.70.0,<2.71.0
   - bioconductor-iranges >=2.36.0,<2.37.0
   - bioconductor-variantannotation >=1.48.0,<1.49.0
   - bioconductor-biocparallel >=1.36.0,<1.37.0

  with channels:
   - file:///opt/host-conda-bld
   - conda-forge
   - bioconda
   - defaults

  The reported errors are:
  - Encountered problems while solving:
  -   - nothing provides requested bioconductor-motifdb >=1.44.0,<1.45.0
  -

  Leaving build/test directories:
    Work:
   /opt/conda/conda-bld/work
    Test:
   /opt/conda/conda-bld/test_tmp
  Leaving build/test environments:
    Test:
  source activate  /opt/conda/conda-bld/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pl
    Build:
  source activate  /opt/conda/conda-bld/_build_env


  Traceback (most recent call last):
    File "/opt/conda/lib/python3.8/site-packages/boa/cli/mambabuild.py", line 141, in mamba_get_install_actions
      solution = solver.solve_for_action(_specs, prefix)
    File "/opt/conda/lib/python3.8/site-packages/boa/core/solver.py", line 244, in solve_for_action
      t = self.solve(specs)
    File "/opt/conda/lib/python3.8/site-packages/boa/core/solver.py", line 234, in solve
      raise RuntimeError("Solver could not find solution."  error_string)
  RuntimeError: Solver could not find solution.Mamba failed to solve:
   - bioconductor-motifdb >=1.44.0,<1.45.0
   - bioconductor-motifstack >=1.46.0,<1.47.0
   - bioconductor-genomicranges >=1.54.0,<1.55.0
   - bioconductor-s4vectors >=0.40.0,<0.41.0
   - bioconductor-bsgenome >=1.70.0,<1.71.0
   - r-tfmpvalue
   - bioconductor-gviz >=1.46.0,<1.47.0
   - bioconductor-rtracklayer >=1.62.0,<1.63.0
   - bioconductor-genomeinfodb >=1.38.0,<1.39.0
   - bioconductor-summarizedexperiment >=1.32.0,<1.33.0
   - r-base 4.3.*
   - r-matrixstats
   - bioconductor-biocgenerics >=0.48.0,<0.49.0
   - r-stringr
   - bioconductor-biostrings >=2.70.0,<2.71.0
   - bioconductor-iranges >=2.36.0,<2.37.0
   - bioconductor-variantannotation >=1.48.0,<1.49.0
   - bioconductor-biocparallel >=1.36.0,<1.37.0

  with channels:
   - file:///opt/host-conda-bld
   - conda-forge
   - bioconda
   - defaults

  The reported errors are:
  - Encountered problems while solving:
  -   - nothing provides requested bioconductor-motifdb >=1.44.0,<1.45.0
  -

  During handling of the above exception, another exception occurred:

  Traceback (most recent call last):
    File "/opt/conda/bin/conda-mambabuild", line 10, in <module>
      sys.exit(main())
    File "/opt/conda/lib/python3.8/site-packages/boa/cli/mambabuild.py", line 256, in main
      call_conda_build(action, config)
    File "/opt/conda/lib/python3.8/site-packages/boa/cli/mambabuild.py", line 228, in call_conda_build
      result = api.build(
    File "/opt/conda/lib/python3.8/site-packages/conda_build/api.py", line 180, in build
      return build_tree(
    File "/opt/conda/lib/python3.8/site-packages/conda_build/build.py", line 3078, in build_tree
      packages_from_this = build(metadata, stats,
    File "/opt/conda/lib/python3.8/site-packages/conda_build/build.py", line 2038, in build
      output_metas = expand_outputs([(m, need_source_download, need_reparse_in_env)])
    File "/opt/conda/lib/python3.8/site-packages/conda_build/render.py", line 787, in expand_outputs
      for (output_dict, m) in deepcopy(_m).get_output_metadata_set(permit_unsatisfiable_variants=False):
    File "/opt/conda/lib/python3.8/site-packages/conda_build/metadata.py", line 2524, in get_output_metadata_set
      conda_packages = finalize_outputs_pass(
    File "/opt/conda/lib/python3.8/site-packages/conda_build/metadata.py", line 884, in finalize_outputs_pass
      fm = finalize_metadata(
    File "/opt/conda/lib/python3.8/site-packages/conda_build/render.py", line 547, in finalize_metadata
      build_unsat, host_unsat = add_upstream_pins(m,
    File "/opt/conda/lib/python3.8/site-packages/conda_build/render.py", line 409, in add_upstream_pins
      host_deps, host_unsat, extra_run_specs_from_host = _read_upstream_pin_files(m, 'host',
    File "/opt/conda/lib/python3.8/site-packages/conda_build/render.py", line 374, in _read_upstream_pin_files
      deps, actions, unsat = get_env_dependencies(m, env, m.config.variant,
    File "/opt/conda/lib/python3.8/site-packages/conda_build/render.py", line 131, in get_env_dependencies
      actions = environ.get_install_actions(tmpdir, tuple(dependencies), env,
    File "/opt/conda/lib/python3.8/site-packages/boa/cli/mambabuild.py", line 150, in mamba_get_install_actions
      raise err
  conda_build.exceptions.DependencyNeedsBuildingError: Unsatisfiable dependencies for platform linux-64: {MatchSpec("bioconductor-motifdb[version='>=1.44.0,<1.45.0']")}
# Last 100 lines of the build log.
